docker/README.md

```diff
@@ -0,0 +1,18 @@
+# PlutoSliderServer Docker Tools
+
+## Building
+
+The most straightforward way of building the slider server docker image is by first cloning this repository, then using the `Dockerfile` included in this subdirectory. Below is an example build command, assuming your PWD is the repository root.
+
+```bash
+docker build -f docker/Dockerfile -t sliderserver .
+```
+
+## Running
+
+There are two recommended bind mounts, one of which is required to run the docker image built in the previous section. The image expects a folder containing notebooks to be served to be mounted to `/etc/sliderserver/notebooks` in the image. Optionally, a `PlutoDeployment.toml` configuration file can be mounted to `/etc/sliderserver/PlutoDeployment.toml` to override the default deployment configuration packaged with the image. Below is an example of a run command, although this will differ greatly depending on deployment use case.
+
+```bash
+docker run -it --rm -v /path/to/notebooks:/etc/sliderserver/notebooks -p 2345:2345 sliderserver
```

Comment:
But it’s on 0.0.0.0 right?

Reply:
Right, by default the `PlutoDeployment.toml` packaged with the image will set the server host to `0.0.0.0` so that connections can come from the host or elsewhere. With it set to its default of `127.0.0.1` I couldn't even access the server from the docker host, which is probably not good behavior for a default configuration.